packages/gatsby-source-wordpress/generate-plugin-options-docs.js

```diff
@@ -0,0 +1,175 @@
+const prettier = require(`prettier`)
```

Comment:
What does this one do? Do we still need it? Do we need to refactor it?

Reply:
Yeah, we need this 👍 this script generates our plugin options docs. This page is generated -> https://github.com/gatsbyjs/gatsby-source-wordpress-experimental/blob/master/docs/plugin-options.md